.editorconfig

```diff
@@ -27,7 +27,7 @@ ij_any_wrap_long_lines = true
 [*.java]
 # Doc: https://youtrack.jetbrains.com/issue/IDEA-170643#focus=streamItem-27-3708697.0-0
 # $ means "static"
-ij_java_imports_layout = $*,|,software.amazon.jdbc.**,|,*,|,java.**,|,javax.**
+ij_java_imports_layout = $*,|,*
```

Comment:
The previous setting conflicted with our google styling so checkstyle would fail. With this change, auto-organizing imports in intellij will correctly order the imports so checkstyle won't complain

Reply:
Does it make sense to just set the layout to whatever our IntelliJ settings are? The auto-imports we use in intelliJ is also custom ordered.

Reply:
Are you referring to the settings in our confluence page (`... import com.*,import java.*,import javax.*,import org.*,import all other imports ...`)? I'm wondering if this ordering would fail of the `all other imports` block had imports that would make the block non-alphabetical. The last sentence of 3.3.3 in [this link](https://google.github.io/styleguide/javaguide.html#s3.3-import-statements) seems to be saying the block needs to be alphabetical but I'm not 100% sure I'm interpreting it correctly